constrained_ik/src/constrained_ik.cpp

```diff
@@ -29,7 +29,7 @@
 #include <constrained_ik/constraint_results.h>
 #include <ros/ros.h>
 
-const std::vector<std::string> SUPPORTED_COLLISION_DETECTORS = {"FCL"}; /**< Supported collision detector */
+const std::vector<std::string> SUPPORTED_COLLISION_DETECTORS = {}; /**< Supported collision detector */
```

Comment:
This change (and other changes to this file) don't seem to belong in this PR?

Reply:
The cartesian_valgrind node failed prior to this change.   It is unrelated to the intent of this PR but since it was a small change I thought it wouldn't hurt to make it.

Reply:
Ok.

It might be good to split these kind of changes into separate commits, so that file & directory layout changes are not mixed with content changes.

I'll leave it to you whether you'd want to change this PR or not.